src/bundle/Resources/public/js/scripts/admin.search.filters.js

```diff
@@ -154,6 +163,76 @@
             defaultDate
         }));
     };
+    const getUsersList = (value) => {
+        const body = JSON.stringify({
+            ViewInput: {
+                identifier: `find-user-by-name-${value}`,
+                public: false,
+                ContentQuery: {
+                    Criteria: {},
+                    FacetBuilders: {},
+                    SortClauses: {},
+                    Query: {
+                        FullTextCriterion: `${value}*`,
+                        ContentTypeIdentifierCriterion: creatorInput.dataset.contentTypeIdentifiers
+                    },
+                    limit: 50,
+                    offset: 0
+                }
+            }
+        });
+        const request = new Request('/api/ezp/v2/views', {
+            method: 'POST',
+            headers: {
+                'Accept': 'application/vnd.ez.api.View+json; version=1.1',
+                'Content-Type': 'application/vnd.ez.api.ViewInput+json; version=1.1',
+                'X-Siteaccess': siteaccess,
+                'X-CSRF-Token': token
+            },
+            body,
+            mode: 'same-origin',
+            credentials: 'same-origin'
+        });
+
+        fetch(request)
+            .then(response => response.json())
+            .then(data => showUsersList(data));
+    };
+    const createUsersListItem = (user) => `<li data-id="${user._id}" data-name="${user.Name}" class="ez-filters__user-item">${user.Name}</li>`;
+    const showUsersList = (data) => {
+        const hits = data.View.Result.searchHits.searchHit;
+        const users = hits.reduce((total, hit) => total + createUsersListItem(hit.value.Content), '');
+
+        usersList.innerHTML = users;
+        usersList.classList.remove('ez-filters__user-list--hidden');
+    };
+    const handleTyping = (event) => {
+        const value = event.target.value;
+
+        window.clearTimeout(getUsersTimeout);
+
+        if (value.length > 2) {
+            getUsersTimeout = window.setTimeout(getUsersList.bind(null, value), 200);
```

Comment:
I think binding is better than create a function to call another function, but maybe it just me ;)

Reply:
There are 2 operations there in every case:
1. case:
  - bind the context and pass the value,
  - call the function.
2. case:
  - define anonymous function,
  - call the required function

In the end it's all about the way it looks. Which looks better: `someFn.bind(null, value)` or `() => someFn(value)`?

Reply:
I know it's the same. For me, it's more obvious and natural to bind than create a function to call a function. But maybe I got used to it before the arrow functions, because you would never wrote something like: `function() {someFn(value)}` :D